submissions/MarharytaBoichenko/oop/index.js

```diff
@@ -0,0 +1,78 @@
+class Inhabitant {
+  constructor(name, species, friends, saying) {
+    this.name = name;
+    this.species = species;
+    this.friends = friends;
+    this.saying = saying;
+    this.properties = ["name", "species", "friends", "saying"];
+  }
+  toPrint() {
+    return this.properties
+      .map((prop) => {
+        const neededProp = this[prop] === undefined ? `no ${prop}` : this[prop];
```

Comment:
Use `Array#filter` to keep props that make sense, then stringify what remains with `Array#map`.

Reply:
ok